include/wlr/render/wlr_renderer.h

```diff
@@ -104,6 +104,12 @@ const struct wlr_drm_format_set *wlr_renderer_get_dmabuf_formats(
 bool wlr_renderer_read_pixels(struct wlr_renderer *r, enum wl_shm_format fmt,
 	uint32_t *flags, uint32_t stride, uint32_t width, uint32_t height,
 	uint32_t src_x, uint32_t src_y, uint32_t dst_x, uint32_t dst_y, void *data);
+
+/**
+ * Blits the dmabuf in src onto the one in dst.
+ */
+bool wlr_renderer_blit_dmabuf(struct wlr_renderer *r,
```

Comment:
Should this function also take a region so that ```zwlr_screencopy_capture_output_region``` can work as well?

Reply:
Could do that, but it could also be added later.

Reply:
Ah right, wlr_renderer is an unstable API, so breaking changes are not an issue.
Maybe a todo entry here as well?

Reply:
Well, I could also add the arguments and just leave them unimplemented. That might be "nicer" thing to do.

Reply:
@emersion, you have an opinion on this?

Reply:
Hmm, I'd prefer to do a breaking change:

1. I don't think a lot of compositors will be using this function, so this would be a "cheap" breaking change
2. It's hard to get the API right when you haven't actually implemented the feature.